common.sh

```diff
@@ -992,3 +992,17 @@ BOAT
   echo -e "${V_VIDOFF}"
   die "$* failed"
 }
+
+setup_qemu_static() {
+  local root_fs_dir="$1"
+  if [[ ${BOARD} == "arm64-usr" && -f "${root_fs_dir}/sbin/ldconfig" ]]; then
```

Comment:
Checking for "/sbin/ldconfig" just ensures that a basic system is already present
in the target directory.